ci/check-release-notes.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python3
+
+import argparse
+import yaml
+import re
+import sys
+
+def validate_release_notes_structure(data, file_name):
```

Comment:
I think I mentioned this before, but this is really asking for having a schema and then just validating against it instead. Not a blocker, but a clear follow up to this I think.

Reply:
@jlebon : so the better approach for this is to have a schema saved and make the yaml file to be verified against the schema each time.